src/view/components/semantic/modules/dropdown/dropdown.ts

```diff
@@ -240,8 +241,13 @@ export class Dropdown implements AfterViewInit, OnDestroy, ControlValueAccessor
    */
   private _applyArrowNavFix($dropdown) {
     let $searchField = $dropdown.children('input.search')
+    let enterEvent = this.enter;
     $searchField.on('keyup', (event:any)=> {
       if (DO_NOT_SEARCH_ON_THESE_KEY_EVENTS[event.keyCode]) {
+        if (event.keyCode == 13 && enterEvent){
```

Comment:
but if we have the cw-input-dropdown tag like this (without the enter parameter), isn't the enter parameter be null?, I wanted this parameter to be optional

  <cw-input-dropdown
      flex
      [value]="bundleStores[0]?.id"
      allowAdditions="true"
      (click)="$event.stopPropagation()"
      (change)="setSelectedBundle($event)">